src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2851,10 +2851,10 @@ private static bool IsAvailableFarEastCodePage(uint codePage)
         private static bool IsAnyDBCSCharSet(uint charSet)
         {
             const uint SHIFTJIS_CHARSET = 128;
-            const uint HANGEUL_CHARSET = 129;
+            const uint HANGUL_CHARSET = 129;
```

Comment:
No, I think we should use `HANGUL_CHARSET` as you suggest in the PR.
I was curious and researched the subject a little bit and just shared my findings.